modules/sfp_keybase.py

```diff
@@ -0,0 +1,203 @@
+            self.results[userName] = True
+
+        content = self.queryUsername(userName)
+
+        if content is None:
+            self.sf.debug("No data found for username")
+            return None
+        
+        evt = SpiderFootEvent("RAW_RIR_DATA", str(content), self.__name__, event)
+        self.notifyListeners(evt) 
+
+        if eventName == "LINKED_URL_EXTERNAL":
+            evt = SpiderFootEvent("USERNAME", str(userName), self.__name__, event)
+            self.notifyListeners(evt)    
+    
+        
+        # Replacing string values that are not enclosed within double quotes
+        # Also replacing values like True and False to their corresponding numeric values
+        # If the above steps aren't performed, json.loads() fails
+
+        # Contains all data about username
+        them = json.loads(str(content.get('them')[0]).replace("'", "\"").replace("True", "1").replace("False", "0"))
+        
+        if them is None or them == "None":
+            self.sf.debug("No data found for username")
+            return None
+
+        # Basic information about the username
+        basics = json.loads(str(them.get('basics')).replace("'", "\""))
+
+        # Profile information about the username
+        profile = json.loads(str(them.get('profile')).replace("'", "\""))
+
+        # Failsafe to prevent reporting any wrongly received data
+        if basics:
+            responseUserName = basics.get('username')
+            if not userName == responseUserName:
+                self.sf.error("Username does not match received response, skipping", False)
+                return None
+        
+        if profile:
+            # Get and report full name of user
+            fullName = profile.get('full_name')
+            if not fullName == "None":
+                evt = SpiderFootEvent("HUMAN_NAME", str(fullName), self.__name__, event)
+                self.notifyListeners(evt)
+            
+            # Get and report location of user
+            location = profile.get('location')
+            if not location == "None":
+                evt = SpiderFootEvent("GEOINFO", str(location), self.__name__, event)
+                self.notifyListeners(evt)
+            
+            # Extract social media information from JSON response
+            socialMediaLinksRegex = ["https:\/\/github.com\/[A-Za-z0-9-_.]+", "https:\/\/twitter.com\/[A-Za-z0-9-_.]+", 
+                "https:\/\/facebook.com\/[A-Za-z0-9-_.]+"]
+            
+            for socialMediaLinkRegex in socialMediaLinksRegex:
+                links = re.findall(socialMediaLinkRegex, str(content))
+                
+                if len(links) == 0:
+                    continue
+                
+                for link in links:
+
+                    if link in self.results:
+                        continue
+                    self.results[link] = True
+
+                    evt = SpiderFootEvent("SOCIAL_MEDIA", str(link), self.__name__, event)
```

Comment:
Look at the format of the `SOCIAL_MEDIA` event from `sfp_social`, as it needs to be the same.